caas/kubernetes/provider/init.go

```diff
@@ -40,6 +41,12 @@ func init() {
 			Provisioner: "kubernetes.io/aws-ebs",
 		},
 	}
+
+	// clusterPreferredOperatorStorage defines the opinionated storage
+	// that Juju requires to be available on supported clusters to
+	// provision storage for operators.
+	// TODO - support regional storage for GCE etc
+	clusterPreferredOperatorStorage = clusterPreferredWorkloadStorage
```

Comment:
we probably should rename `clusterPreferredOperatorStorage` to `jujuPreferredStorage` for both app `operator` and `controller` pod later.

Reply:
yeah, ok, will do that before landing